modules/machine-health-checks-about.adoc

```diff
@@ -27,6 +27,8 @@ Consider the timeouts carefully, accounting for workloads and requirements.
 * Too short timeouts can result in a remediation loop. For example, the timeout for checking the `NotReady` status must be long enough to allow the machine to complete the startup process.
 ====
 
+If you have configured the `MachineHealthCheck` resource, ensure that you stop it before updating the cluster.
```

Comment:
Does this apply to only MHCs that the user created? It appears that there is a MHC installed by default (at least in 4.8)

```
oc get MachineHealthCheck --all-namespaces
NAMESPACE               NAME                              MAXUNHEALTHY   EXPECTEDMACHINES   CURRENTHEALTHY
openshift-machine-api   machine-api-termination-handler   100%           0                  0

```
Does the user need to remove this one?

Reply:
s/configured/deployed or created to match the [rest of the assembly](https://docs.openshift.com/container-platform/4.8/machine_management/deploying-machine-health-checks.html).

Reply:
@beekhof and @n1r1 - Request you to review and confirm if the user needs to remove the MachineHealthCheck that @mburke5678 has pointed out. Thanks!

Reply:
that's a good point. 
I don't remember the purpose of `machine-api-termination-handler`. something with cloud env? spot instances?
@JoelSpeed ?

Reply:
`machine-api-termination-handler` is installed by CVO so you won't be able to remove it (well you can but it'll be recreated immediately). It enables the spot instance termination notices to be observed on AWS, GCP and Azure.

What are the semantics here around stopping an MHC? We have a pause annotation now, is that what we are planning to use rather than remove/recreate?

Reply:
@JoelSpeed I believe that the pause mechanism is being introduced in 4.9 by @slintes , whereas this ticket addressed cluster updates in 4.8. Therefore, for 4.8, the recommendation is to stop (remove) MHC.

Reply:
At the moment this PR is going into the main docs, so would also be included in 4.9 as far as I can tell.
Does it not make sense to get the docs right for 4.9 then work backwards? 
If we merge this as is, this will be present in the 4.9 docs at which point it is not up to date and we can do better

Reply:
@JoelSpeed I understand you point, however, I'd like to direct you to https://github.com/openshift/openshift-docs/pull/35373#issuecomment-896317008.

When we address [TELCODOCS-291](https://issues.redhat.com/browse/TELCODOCS-291) in 4.9, it will take care of your concern and the updates will be available in their relevant release streams then.

Reply:
Ack, going back to the original question then, no, you won't be able to remove the `machine-api-termination-handler` MHC, this will need to be noted in the docs IMO. You probably want to highlight something like removing "user created" MHCs rather than all MHCs

Reply:
@JoelSpeed Thank you. Re-writing it as follows. Kindly review:
 While a `MachineHealthCheck` resource deployed by default (such as `machine-api-termination-handler`) cannot be removed, ensure that you remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.

Reply:
@JoelSpeed and @mburke5678 : I have updated the note to address you comments. Request you to kindly review the latest content. Thanks!

Reply:
@abhatt-rh Can I make one last suggestion? In the About the OpenShift update service topic, maybe swap the [two sentences in the note](https://github.com/openshift/openshift-docs/pull/35373/files#diff-769155bafe6ff5307aa01d79f6b07ad34bc72fd8431fef3501a554802d653ee5R50)? 
At that point, as a user I might not know why I need to delete the MHCs.  (And maybe a phrase at the end so that users do not get confused when the MHC they deleted comes back [like I did!]). Just a thought.

If you have deployed a MachineHealthCheck resource, ensure that you remove it before updating the cluster.
However, a MachineHealthCheck resource deployed by default (such as machine-api-termination-handler) cannot be removed and will be recreated.

Reply:
@mburke5678 Yes, I understand and that makes sense about the OSUS topic. I will implement it. Thank you